source/stylesheets/_grid.scss

```diff
@@ -28,18 +28,12 @@ body {
   // @include grid-visual;
   @include grid-container;
   @extend .grid-container--flex;
-  @media screen and (max-width: 1200px) {
+  @media screen and (max-width: $max-viewport) {
     padding: 0 20px;
```

Comment:
responsive fix. adds the side padding before elements bump the edges of the viewport